wire-library/wire-runtime-swift/src/main/swift/CodingUserInfoKey+Wire.swift

```diff
@@ -29,16 +29,6 @@ extension JSONDecoder {
         /// - Note: For dictionaries, it is necessary to wrap it in `@ProtoMapEnumValues`
         case returnNil
     }
-
-    /// The decoding strategy to use for StringEncoded types that are themselves Decodable
-    /// Defaults to .disallowRawDecoding
-    /// - Note: ProtoMap Dictionary keys are always decoded as strings
-    public enum StringEncodedDecodingStrategy {
```

Comment:
I noticed this while re-reading the proto3<->json document.
Raw numbers are supposed to be accepted